scripts/require-sync/go.mod

```diff
@@ -0,0 +1,111 @@
+module require-sync
+
+go 1.19
+
+// Prevent otel dependencies from getting out of sync.
+replace (
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.20.0
```

Comment:
do we need the otel dependencies?

Reply:
I believe these will be required till this issue is resolved: https://github.com/open-policy-agent/frameworks/issues/210